inference-engine/src/inference_engine/src/cnn_network_ngraph_impl.cpp

```diff
@@ -115,12 +116,17 @@ CNNNetworkNGraphImpl::CNNNetworkNGraphImpl(const std::shared_ptr<Function>& nGra
     : _ngraph_function(nGraph),
       _ie_extensions(exts),
       _new_api(newAPI) {
+    int64_t version = 11;
+    if (_ngraph_function->get_rt_info().count("version")) {
+        version = std::dynamic_pointer_cast<ov::VariantWrapper<int64_t>>(_ngraph_function->get_rt_info().at("version"))
+                      ->get();
+    }
     // Restore usual attributes for CNNNetwork
     auto keep_input_info = [=](CNNNetworkNGraphImpl& network, const DataPtr& inData) {
         InputInfo::Ptr info(new InputInfo());
         info->setInputData(inData);
 
-        if (!_new_api) {
+        if (!_new_api || version == 10) {
```

Comment:
why do we need it?
For IR v10 and new API we have https://github.com/openvinotoolkit/openvino/pull/7853 which inserts precision conversions as a part of the graph. In that PR I also remove IR version from rt_info as we agreed